anjumol/react_new/section_9/src/components/NoProjectSelected.jsx

```diff
@@ -0,0 +1,23 @@
+import noProjectImage from '../assets/no-projects.png';
```

Comment:
images are duplicate in public and assets folder we can  remove the  assets folder 
import image mentioned below
import noProjectImage from '../../public/logo.png';